### PR TITLE
Corrected snippets in `labs/Demos/Cryptol`

### PR DESCRIPTION
**Pull request by WeeknightMVP**
This PR corrects snippets in `labs/Demos` based on running the experimental [Markdown Snippet Extractor](https://github.com/weaversa/cryptol-course/pull/111).

### CODE REVIEW

**Comment by WeeknightMVP** [comment]
We should revisit the wording in these and other modules, per Issue #104.

File: labs/Demos/Cryptol/Caesar.md [hunk 1]
```diff
@@ -59,6 +61,7 @@ caesar msg = map rot3 msg
 ```
 
 ```shell
+labs::Demos::Cryptol::Caesar> :s prover=abc
```
Comment (weaversa) [inline labs/Demos/Cryptol/Caesar.md:64]: Let’s stick with z3 everywhere we can. Not everyone can use other provers.

File: labs/Demos/Cryptol/OneTimePad.md [hunk 2]
```diff
@@ -269,9 +271,10 @@ reasonably have been "GOODBYE". Then Eve can exploit Cryptol to
 deduce the pre-shared key Bob just used:
 
 ```shell
-labs::Demos::Cryptol::OneTimePad> :sat \psk -> (encrypt psk pt2) == ct2
-(\psk -> (encrypt (psk : String 7) pt2) == ct2) "ZUGESAG" = True
-(Total Elapsed Time: 0.042s, using Z3)
+labs::Demos::Cryptol::OneTimePad> :sat \psk2 -> (encrypt`{7} psk2 pt2) == ct2
+Satisfiable
```
Comment (weaversa) [inline labs/Demos/Cryptol/OneTimePad.md:275]: I’m confused why this line prints the word Satisfiable but line 294 doesn’t.

File: labs/Demos/Cryptol/Caesar.md (hunk 1 above)
Comment (WeeknightMVP) [reply labs/Demos/Cryptol/Caesar.md:64]: Well when I wrote this, Z3's performance was tolerable...but when I ran it last night the `recovery` proofs were especially slow.  Perhaps we could comment this line out and just run it for testing/CI.

File: labs/Demos/Cryptol/OneTimePad.md (hunk 2 above)
Comment (WeeknightMVP) [reply labs/Demos/Cryptol/OneTimePad.md]: That was an error on my part.  When running a Cryptol module in batch mode, but not when running the interpreter interactively, `:sat` commands show `Testing...` and `Satisfiable`.

File: labs/Demos/Cryptol/Caesar.md (hunk 1 above)
Comment (weaversa) [reply labs/Demos/Cryptol/Caesar.md:64]: How about adding a note that `z3` works but is slow, so we're demonstrating with `abc`, and folks can use whichever solver they have installed?

**Pull request merged**